stylus/generic/typography.styl

```diff
@@ -55,17 +55,17 @@ $caption
     color coolGrey
 
 /*------------------------------------*\
-  Alignement
+  Link
 \*------------------------------------*/
+$link
+    &:link
+        color dodgerBlue
+        text-decoration none
 
-$left
```

Comment:
All this part about text alignment was already a duplicate of the code in `text.styl`, this is just a clean up don't worry ! ;)